_reports/akshatkarani.md

```diff
@@ -0,0 +1,149 @@
+  - Bonding : https://gitlab.com/coala/GSoC/gsoc-2019/-/milestones/19
+  - Phase 1 : https://gitlab.com/coala/GSoC/gsoc-2019/-/milestones/20
+  - Phase 2 : https://gitlab.com/coala/GSoC/gsoc-2019/-/milestones/21
+  - Phase 3 : https://gitlab.com/coala/GSoC/gsoc-2019/-/milestones/22
+bio: >
+ I'm a third year student of Computer Science & Engineering at
+ Indian Institute of Technology, Dharwad. I am enthusiatic about
+ contributing to open source and working on my own projects.
+ I participated in GSoC and worked with [coala](https://coala.io) to provide
+ support for bears to define their own custom actions. I also provided support
+ for bears to suggest multiple patches for a problem.
+social:
+  - GitHub:
+      - username: akshatkarani
+      - link: https://github.com/akshatkarani
+  - GitLab:
+      - username: akshatkarani
+      - link: https://gitlab.com/akshatkarani
+  - Gitter:
+      - username: akshatkarani
+      - link: https://gitter.im/akshatkarani
+  - Openhub:
+      - username: akshatkarani
+      - link: https://www.openhub.net/accounts/akshatkarani
+email: akshatkarani@gmail.com
+blog: http://akshatkarani.github.io/blog
+activity:
+ - 1:
+   - repo: cEP
+   - link: https://github.com/coala/cEPs/pull/182/
+   - details: >
+      cEP for the Project
+ - 2:
+   - repo: coala
+   - link: https://github.com/coala/coala/pull/6039
+   - details: >
+       Fixing color_letter function in ConsoleInteraction module
+ - 3:
+   - repo: coala
+   - link: https://github.com/coala/coala/pull/6029
+   - details: >
+      Adding actions attribute to Result class
+ - 4:
+   - repo: coala-bears
+   - link: https://github.com/coala/coala-bears/pull/2927
+   - details: >
+      EditCommitMessageAction and AddNewlineAction for GitCommitBear
+ - 5:
+   - repo: coala
+   - link: https://github.com/coala/coala/pull/6046
+   - details: >
+      Tutorial for writing bear specific actions
+ - 6:
+   - repo: coala
+   - link: https://github.com/coala/coala/pull/6057
+   - details: >
+      Implementing AlternatePatchAction and support for bears to provide multiple patches
+ - 7:
+   - repo: coala-bears
+   - link: https://github.com/coala/coala-bears/pull/2947
+   - details: >
+      DeleteFileAction for DuplicateFileBear
+ - 8:
+   - repo: coala-bears
+   - link: https://github.com/coala/coala-bears/pull/2948
+   - details: >
+      Providing multiple patches for FilenameBear
+ - 9:
+   - repo: coala
+   - link: https://github.com/coala/coala/pull/6060
+   - details: >
+      Update the docs to explain how bears can suggest multiple patches
+---
+
+### Next generation action system
+
+
+#### Work Done
+
+- Made some changes to `action_dict`, now the key of the dictionary
+is `id` of the object rather that name of the class.
+
+- Updated the `color_letter` function in ConsoleInteraction module to display
+the action on console properly.
+
+- I added two new attribute to `Result` class:
+
+   `actions` attribute which is a list of action instances and
```

Comment:
Should be nested numbered points. See how https://deploy-preview-747--coala-projects.netlify.com/gsoc/2019/statusreport/akshatkarani looks.

Reply:
Done